### PR TITLE
Dispatcher service

### LINKED ISSUE #67: [Feature Request] Dispatcher Service
Would it be possible to implement the Dispatcher Service (https://docs.librenms.org/Extensions/Dispatcher-Service/) into the Docker?

### PR DESCRIPTION
**Pull request by crazy-max**
Closes #67

### CODE REVIEW

File: doc/docker/environment-variables.md [hunk 1]
```diff
@@ -12,19 +12,32 @@
...
-* `LIBRENMS_DISTRIBUTED_POLLER_NAME`: Optional name of poller (defaults to hostname)
-* `LIBRENMS_DISTRIBUTED_POLLER_GROUP`: By default, all hosts are shared and have the poller_group = 0. To pin a device to a poller, set it to a value greater than 0 and set the same value here. One can also specify a comma separated string of poller groups. The poller will then poll devices from any of the groups listed. [docs](https://docs.librenms.org/#Extensions/Distributed-Poller/#distributed-poller)
-* `LIBRENMS_DISTRIBUTED_POLLER_MEMCACHED_HOST`: Memcached server for poller synchronization (Defaults to `$MEMCACHED_HOST`)
-* `LIBRENMS_DISTRIBUTED_POLLER_MEMCACHED_PORT`: Port of memcached server (Defaults to `$MEMCACHED_PORT`)
+> :warning: Only used if you enable and run a [sidecar dispatcher container](../notes/crons.md)
+
+* `SIDECAR_DISPATCHER`: Set to `1` to enable sidecar dispatcher mode for this container (default `0`)
+* `LIBRENMS_SERVICE_POLLER_WORKERS`: Processes spawned for polling (default `24`)
+* `LIBRENMS_SERVICE_SERVICES_WORKERS`: Processes spawned for service polling (default `8`)
+* `LIBRENMS_SERVICE_DISCOVERY_WORKERS`: Processes spawned for discovery (default `16`)
```
Comment (murrant) [inline doc/docker/environment-variables.md:22]: I plan to make the worker settings configurable via the user interface (per poller node)

**Comment by murrant** [comment]
poller-service.py is broken and should not be used.
The dispatcher service can take can supplant all of them.  Unfortunately, memcached is still needed because daily.php uses it for locking.  This should be updated at some point.

**Comment by crazy-max** [comment]
@murrant

> poller-service.py is broken and should not be used.

`poller-service.py` is not used in the Docker image, only the `poller-wrapper.py` for now. What do you mean about that?

**Comment by murrant** [comment]
I was just answering your question on the previous issue.

**Pull request merged**